internal/databases/postgres/backup.go

```diff
@@ -235,7 +235,14 @@ func setTablespacePaths(spec TablespaceSpec) error {
 	for _, location := range spec.tablespaceLocations() {
 		err := fs.NewFolder(location.Location, "").EnsureExists()
 		if err != nil {
-			return fmt.Errorf("error creating folder for tablespace %v", err)
+			// Don't show error when correct symlink already exist
```

Comment:
it would be very good to do this check within EnsureExists() function.

Reply:
EnsureExists() used for Folder type. 
I think, I should not create a new type to work with symbolic links if only a few calls are needed.

Reply:
You can add this logic to the [TablespaceLocation](https://github.com/wal-g/wal-g/blob/7f699a029a4d625c18136ebf9e9d87b022c27022/internal/databases/postgres/tablespace_spec.go#L28) type